store.go

```diff
@@ -600,6 +601,14 @@ func GetStore(options StoreOptions) (Store, error) {
 		}
 	}
 
+	if onTmpfs, err := mount.IsOnVolatileStorage(options.RunRoot); err != nil || !onTmpfs {
+		if err != nil {
+			return nil, errors.Wrapf(err, "cannot check if %s is on tmpfs", options.RunRoot)
+		} else {
+			return nil, errors.Wrapf(ErrTargetNotVolatile, "%s must be on tmpfs", options.RunRoot)
+		}
+	}
+
```

Comment:
Do we also need the check for `ReloadConfigurationFile(...)` (and extend it to return an error)?

Reply:
yes good idea.  Pushed a new version